extensions/dub/package.json

```diff
@@ -47,6 +48,38 @@
       "mode": "view"
     }
   ],
+  "tools": [
+    {
+      "name": "getAllShortLinks",
+      "title": "Get All Short Links",
+      "description": "Get all shortened links from dub.co"
+    },
+    {
+      "name": "getAllDomains",
+      "title": "Get All Domains",
+      "description": "Get all domains from dub.co"
+    },
+    {
+      "name": "createShortLink",
+      "title": "Create Short Link",
+      "description": "Create a new shortened link with dub.co"
+    },
+    {
+      "name": "deleteShortLink",
+      "title": "Delete Short Link",
+      "description": "Delete a shortened link from dub.co"
+    },
+    {
+      "name": "getAllTags",
+      "title": "Get All Tags",
+      "description": "Get all tags from dub.co"
+    },
+    {
+      "name": "getShortLinksCount",
+      "title": "Get Short Links Count",
+      "description": "Get the number of shortened links in dub.co"
+    }
+  ],
```

Comment:
Add some [evals](https://developers.raycast.com/ai/write-evals-for-your-ai-extension) to test the AI extension. Also, inform users about the prompts they can ask.
```
"ai": {
  "evals": [
    {
      "input" : "@dub Get all short links",
      "mocks" : {
        "getAllShortLinks" : []
      },
      "expected" : [
        {
          "callsTool" : {
            "arguments" : {},
            "name" : "getAllShortLinks"
          }
        }
      ]
    }
  ]
},
```
![Raycast 2025-03-13 at 21 56 54](https://github.com/user-attachments/assets/734eb037-3b1d-4b30-a254-7c2a61522366)